sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log azure-ai-formrecognizer
 
+## 1.0.0b4 (Unreleased)
```

Comment:
Everything looks good, you just forgot to remove mention of RecognizedReceipt from the readme: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/formrecognizer/azure-ai-formrecognizer#formrecognizerclient